bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -104,7 +104,15 @@ Compare the following output with the right picture at the beginning of the arti
                                     41451                                     
 
 ```
-         
+
+## State
+
+The state channel shows an image of the panels on the wall. You have to configure things for each panel to
```

Comment:
Please reformat the text so that there's once sentence per line. This will make it easier to diff/comment etc.

Reply:
Thanks! Fixed.